test/config_test.py

```diff
@@ -41,7 +41,9 @@ def setup_method(self):
         self.safe_copy = os.environ.copy()
 
     def teardown_method(self):
-        os.environ = self.safe_copy
+        os.environ.clear()
+        for prop in self.safe_copy:
+            os.environ[prop] = self.safe_copy[prop]
```

Comment:
This is a good change, but we can actually remove both the `setup_method` and `teardown_method` for this class. This code in conftest.py does this for every test, not just this class (`autouse=True` is the magic that makes that fixture executed for every test):

```python
@pytest.fixture(autouse=True)
def environ():
    """Make sure tests don't clobber the environment"""
    safe_copy = os.environ.copy()
    try:
        yield
    finally:
        os.environ.clear()
        os.environ.update(safe_copy)
```

Reply:
If I remove the setup_method and teardown_method Lint has an error and the test won't run.

Reply:
What's the error?